AnnoyancesFilter/sections/popups.txt

```diff
@@ -101,6 +101,8 @@ sodapdf.com##.be-tooltip
 eastbaytimes.com#$#.modal-scrollable { display: none!important; }
 eastbaytimes.com#$#.connext-modal-backdrop { display: none!important; }
 eastbaytimes.com#$#body { overflow: visible!important; }
+protocol.com#$##form_out { display: none !important; }
```

Comment:
does not work with apps(`display: block;`)

`protocol.com#$#body { overflow: visible !important; }`
better `auto`
`protocol.com#$#body div#form_out { display: none !important; }`
`protocol.com#$#body { overflow: auto !important; }`

Reply:
Actually both works on my end.